### DISCUSSION TITLE
where is  rabbitmq_server-3.8.8 delayed-message-exchange

### DISCUSSION THREAD

**Discussion by yujunchengg** [rabbitmq/rabbitmq-delayed-message-exchange#206] [Q&A]
my rabbitmq-server version is 3.8.8,  I want to install the same version delayed-message-exchange

**Comment by jayendrapawar** [comment]
I want to install specific version in MacOS terminal of rabbitMQ 3.10.x , It is compatible with delayed-message-exchange, latest version 3.11.x is not compatible

**Reply by lukebakken** [reply]
https://github.com/rabbitmq/rabbitmq-delayed-message-exchange/releases/tag/3.10.2
